source/Octopus.Cli/CliProgram.cs

```diff
@@ -154,7 +154,7 @@ static int PrintError(Exception ex)
                 Log.Error(ex.Message);
                 if (LogExtensions.IsKnownEnvironment())
                 {
-                    Log.Error("This error is most likely occurring while executing Octo.exe as part of an automated build process. The following doc is recommended to get some tips on how to troubleshoot this: https://g.octopushq.com/OctoexeTroubleshooting");
+                    Log.Error("This error is most likely occurring while executing octo.exe as part of an automated build process. The following doc is recommended to get some tips on how to troubleshoot this: https://g.octopushq.com/OctoexeTroubleshooting");
```

Comment:
Can we figure out what the executing assembly's name is?

Reply:
Will also need to create a new octopurl for that link that doesn't include `octoexe`

Reply:
Good idea. Fixed 6 instances.

Reply:
I've created a card for updating docs, and then we can do a pass over octopurls and other references to docs.